Demo/Demo/FeatureViewControllers/PaymentButtonCustomizationViewController.swift

```diff
@@ -125,6 +133,9 @@ class PaymentButtonCustomizationViewController: UIViewController {
         if sender == fundingPicker {
             reloadColorPicker()
         }
+        let fundingSource = PaymentButtonFundingSource.allCases[fundingPicker.selectedSegmentIndex]
+        labelPicker.isHidden = fundingSource != .payPal
```

Comment:
It looks like certain labels are only available with certain sizes. Is this expected (I assume so just want to confirm)? If so, we can probably add something like this under `fundingSource` here to only display the labels when they are applicable:
```
let sizeSource = PaymentButtonSize.allCases[sizePicker.selectedSegmentIndex]
labelPicker.isHidden = fundingSource != .payPal || sizeSource == .mini || sizeSource == .collapsed
```
Can likely do the same in the SwiftUI Demo. Also I noticed that all of the labels apply as expected to the `expanded` and `full` size buttons _except_  `payWith` on the `expanded` button. Is this intentional?

Reply:
@jaxdesmarais it is. This is what we copied from NXO. Im guessing it has to do with the logo, because the small ones only have the PayPal logo with the `P`, so it doesnt make sense to add a label, it would look something like `P` `Checkout`, its not readable. 
Same with `payWith`. This reasons are all speculative though, because we never had requirements.

Reply:
Yeah, I understand why they don't display on the smaller buttons, we should probably call that out somewhere so we don't get inbounds about it.

What I was not sure if is still why `payWith` doesn't display on the expanded button. All of the other labels display as expected on `expanded` and `full` so it seems odd to me that one does not. Again may be expected but worth confirming with NXO if this is a bug or not.

Reply:
@jaxdesmarais code wise it is expected. They also did a major refactor of the buttons not so long ago, i havent tested it fully, so we might have diverged from them now. I'll see if i can add it to `expanded` for consistencies sake

Reply:
@jaxdesmarais i updated the pickers on both demos, and also added `payLater` to the `expanded size as well. Also added it to the docs